astropy/cosmology/connect.py

```diff
@@ -119,10 +126,10 @@ class CosmologyWrite(io_registry.UnifiedReadWrite):
     -----
     """
 
-    def __init__(self, instance, cls):
+    def __init__(self, instance: Cosmology, cls: type[Cosmology]) -> None:
         super().__init__(instance, cls, "write", registry=readwrite_registry)
 
-    def __call__(self, *args, **kwargs):
+    def __call__(self, *args: Any, **kwargs: Any) -> None:
         self.registry.write(self._instance, *args, **kwargs)
```

Comment:
All the classes here inherit from `io_registry.UnifiedIORegistry()` and they all implement only `__init__()` and `__call__()`. Why is it then that this wrapper for `io_registry.UnifiedIORegistry().registry.write()` returns `None`...

Reply:
The writers do not return anything. They write to disk.